### ISSUE TITLE
PropertyFieldCollectionData - 1.13.0 - 'Each child in an array or iterator should have a unique "key" prop' on first WP configuration

### ISSUE THREAD

**Issue by FranckyC** [SharePoint/sp-dev-fx-property-controls#128]
After installing the new released 1.13.0 version, the sorting seems to break (see screenshot). Using the beta version was fine.

#### Category
- [ ] Enhancement
- [ X] Bug
- [ ] Question

#### Version

Please specify what version of the library you are using: [1.13.0]

The property pane property is configured as follow (with enableSorting):

```
PropertyFieldCollectionData('resultTypes', {
                manageBtnLabel: strings.ResultTypes.EditResultTypesLabel,
                key: 'resultTypes',
                panelHeader: strings.ResultTypes.EditResultTypesLabel,
                panelDescription: strings.ResultTypes.ResultTypesDescription,
                enableSorting: true,
                label: strings.ResultTypes.ResultTypeslabel,
                value: this.properties.resultTypes,
                fields: [
                    ...

```

![image](https://user-images.githubusercontent.com/5659122/49958328-768ad080-fed8-11e8-9826-3de0ceb996f9.png)

**Comment by FranckyC** [comment]
This error only appears during the first instantiation.

**Comment by estruyf** [comment]
@FranckyC that's indeed weird, will check out what's going on, and release a bug fix.

**Comment by estruyf** [comment]
It seems to only occur in the local workbench. Just released `v1.13.1` as a beta with the included fix.
